lib/cocoapods-core/specification/dsl.rb

```diff
@@ -76,8 +76,10 @@ module DSL
       #   @param  [String] name
       #           the name of the pod.
       #
-      root_attribute :name,
-                     :required => true
+      attribute :name,
```

Comment:
:+1: in that case, need to validate that no `/` gets into `attributes_hash[:name]`